pymc3/backends/sqlite.py

```diff
@@ -366,14 +366,14 @@ def _get_table_list(cursor):
 
 
 def _get_var_strs(cursor, varname):
-    cursor.execute('SELECT * FROM [{}]'.format(varname))
+    cursor.execute(f'SELECT * FROM [{varname}]')
```

Comment:
boy howdy we should get rid of this backend

Reply:
yes...